### PR TITLE
new file from NadiiaUrban branch

### PR DESCRIPTION
**Pull request by NadiiaUrban**
Please check it

### CODE REVIEW

File: src/NadiiaUrban/Task1.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package NadiiaUrban;
+
+public class Task1 {
+    //Hello World
```
Comment (morb1k) [inline src/NadiiaUrban/Task1.java:4]: enough to get a job

**Review by morb1k — CHANGES_REQUESTED** [review]
add main method and print statement

**Comment by NadiiaUrban** [comment]
please check it one more time

**Review by morb1k — APPROVED** [review]
Great. 200k$ daily

**Pull request merged**